src/storage/store/node_table.cpp

```diff
@@ -26,6 +26,13 @@ void NodeTable::initializeData(NodeTableSchema* nodeTableSchema) {
     }
 }
 
+void NodeTable::resetColumns(catalog::NodeTableSchema* nodeTableSchema) {
```

Comment:
Why do we need this ?

Reply:
This function is used to close the files for nodeTable, if we don't close the files, we can't do the rename